Resources/Prototypes/Corvax/Reagents/Materials/materials.yml

```
@@ -0,0 +1,8 @@
+- type: material
+  id: SheetPrinter
+  stackEntity: SheetPrinter1
+  name: materials-paper
+  icon: { sprite: Objects/Misc/bureaucracy.rsi, state: paper }
+  color: "#ffffff"
+  price: 0.05
+  
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)